creator-node/src/services/stateMachineManager/stateMonitoring/stateMonitoringUtils.js

```diff
@@ -0,0 +1,409 @@
+const _ = require('lodash')
```

Comment:
this file was renamed from `stateMachineManager/monitoring/utils.js` and changed to directly export functions instead of a class. no other changes besides that so feel free to skip or skim this one